docs/userguides/projects.md

```diff
@@ -122,8 +122,8 @@ installed, giving you more flexibility in how you define your scripts.
 
 
 Use tests to verify your project. Testing is a complex topic, learn more about testing using Ape framework
-[here](docs/userguides/testing.md)
+[here](../docs/userguides/testing.md)
```

Comment:
Have to confirm docs still build correctly here

Reply:
yea apologies, it should just be "./testing.md" instead of "docs/userguides/testing.md"